akka-http-core/src/main/java/akka/http/javadsl/model/headers/TlsSessionInfo.java

```diff
@@ -0,0 +1,22 @@
+package akka.http.javadsl.model.headers;
+
+import javax.net.ssl.SSLSession;
+
+/**
+ * Model for the synthetic `Tls-Session-Info` header which carries the SSLSession of the connection
+ * the message carrying this header was received with.
+ *
+ * This header will only be added if it enabled in the configuration by setting
+ *
+ *     akka.http.(client|server).parsing.tls-session-info-header = on
```

Comment:
This indent causes it to render as <pre/>? Never seen that before hm...

I thought it's:

```
'''
akka.http.(client|server).parsing.tls-session-info-header = on
'''
```

We usually use `[client|server]`.